bot/code_review_bot/report/base.py

```diff
@@ -37,7 +37,7 @@
 You can view these defects on [the code-review frontend]({frontend_url}) and on [Treeherder]({treeherder_url}).
 """
 COMMENT_LINK_TO_DOC = """
-You have touched the documentation, you can find it rendered [here]({link_to_doc}) for a week.
+You have touched the documentation, you can find it rendered [here]({link_to_doc}) of diff {diff_id} for a week.
```

Comment:
```suggestion
You have touched the documentation in diff {diff_id}, you can find it rendered [here]({link_to_doc}) for a week.
```

Reply:
Now I know the reason why i have to take IELTS exam xD cc @sylvestre

Reply:
huhu :)